src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -2255,7 +2255,7 @@ int xcldev::device::testM2m()
     if (dev == nullptr)
         return -EINVAL;
 
-    dev->sysfs_get<int>("", "kds_numcdmas", errmsg, m2m_enabled, 0);
+    dev->sysfs_get<int>("", "m2m", errmsg, m2m_enabled, 0);
```

Comment:
Thank you updated the code in this codebase.  Please do note that it will be removed from the code base shortly.

Reply:
Makes sense.